src/components/timeline/index.js

```diff
@@ -203,6 +203,14 @@ class TimelineSettingsActiveTabView extends React.PureComponent<{|
 }
 
 class Timeline extends React.PureComponent<Props> {
+  state = {
```

Comment:
I think this state needs to have a type provided for it. e.g.

`class Timeline extends React.PureComponent<Props, State>`

Reply:
You are seeing my first commit.  The  second commit has that fixed